tests/sentry/integrations/slack/test_disable.py

```diff
@@ -21,6 +22,18 @@
 secret = "hush-hush-im-invisible"
 
 
+def raiseStatusFalse():
```

Comment:
nit: can you make this `raise_status_false`?

Reply:
I actually didn't end up using these mocks so I'll take this out